07-routing-gossip.md

```diff
@@ -28,49 +34,50 @@ The willingness of the initiating node to announce the channel is signaled durin
 
 ### Requirements
 
-If the `open_channel` message had the `announce_channel` bit set, then both nodes must send the `announcement_signatures` message, otherwise they MUST NOT.
-
-If sent, `announcement_signatures` messages MUST NOT be sent until `funding_locked` has been sent and the funding transaction has at least 6 confirmations.
+The `announcement_signatures` message is created by constructing a `channel_announcement` message corresponding to the newly established channel and signing it with the secrets matching an endpoint's `node_id` and `bitcoin_key`. The message is then sent using an `announcement_signatures`.
 
 The `short_channel_id` is the unique description of the funding transaction.
-It is constructed with the most significant 3 bytes as the block
+It is constructed with the most significant 3 bytes indicating the block
 height, the next 3 bytes indicating the transaction index within the
 block, and the least significant two bytes indicating the output
-index which pays to the channel.
+index that pays to the channel.
+
+If the `open_channel` message had the `announce_channel` bit set, then both nodes must send the `announcement_signatures` message, otherwise they MUST NOT.
+
+If sent, `announcement_signatures` messages MUST NOT be sent until `funding_locked` has been sent and the funding transaction has at least 6 confirmations.
 
-The `announcement_signatures` message is created by constructing a `channel_announcement` message corresponding to the newly established channel, and sign it with the secrets matching their `node_id` and `bitcoin_key`, and send them using an `announcement_signatures`.
 The recipient MAY fail the channel if the `node_signature` or `bitcoin_signature` is incorrect.
 The recipient SHOULD queue the `channel_announcement` message for its peers if it has sent and received a valid `announcement_signatures` message.
 
 On reconnection, a node SHOULD retransmit the `announcement_signatures` message if it has not received an `announcement_signatures` message, and MUST respond to the first `announcement_signatures` message after reconnection with its own `announcement_signatures` message.
 
-## The `channel_announcement` message
+## The `channel_announcement` Message
 
-This message contains ownership information about a channel.  It ties
-each on-chain Bitcoin key to the lightning node key, and vice-versa.
+This message contains ownership information about a channel. It ties
+each on-chain Bitcoin key to the Lightning node key, and vice-versa.
 The channel is not really usable until at least one side has announced
 its fee levels and expiry using `channel_update`.
 
-In order to prove the existence of channel between `node_1` and
-`node_2` we need to:
+Proving the existence of a channel between `node_1` and
+`node_2` requires:
 
-1. Prove that the funding transaction pays to `bitcoin_key_1` and
+1. Proving that the funding transaction pays to `bitcoin_key_1` and
```

Comment:
proving...`bitcoin_key_1`  <--(no 'and')

Reply:
The whole line is actually: "1. Proving that the funding transaction pays to `bitcoin_key_1` and
   `bitcoin_key_2`."Unless there's a style I'm missing, kept.